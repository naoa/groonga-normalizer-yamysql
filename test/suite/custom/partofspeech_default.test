register normalizers/yamysql

normalize NormalizerYaMySQLPartofspeech "これはテストです。" WITH_TYPES|WITH_CHECKS

