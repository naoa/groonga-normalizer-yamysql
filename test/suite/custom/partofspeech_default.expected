register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQLPartofspeech "これはテストです。" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "これテスト",
    "types": [
      "hiragana",
      "unknown",
      "katakana",
      "katakana",
      "unknown"
    ],
    "checks": [
      3,
      0,
      0,
      3,
      0,
      0,
      6,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
